tools/monorepo-utils/src/changefile/lib/__tests__/github.ts

```diff
@@ -131,6 +131,9 @@ describe( 'getChangelogSignificance', () => {
 			'#### Message ' +
 			'<!-- Add a changelog message here -->\r\n' +
 			'This is a very useful fix.\r\n' +
+			'</details>\r\n' +
+			'<details>\r\n' +
+			'<summary>Changelog Entry Comment</summary>' +
```

Comment:
The template was updated in https://github.com/woocommerce/woocommerce/pull/48123, but the tests were never updated as well.